easybuild/easyconfigs/d/deepTools/deepTools-3.5.5-foss-2022b.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'PythonBundle'
+
+name = 'deepTools'
+version = '3.5.5'
+
+homepage = 'https://deeptools.readthedocs.io/'
+description = """deepTools is a suite of python tools particularly developed for the efficient analysis of
+ high-throughput sequencing data, such as ChIP-seq, RNA-seq or MNase-seq."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('SciPy-bundle', '2023.02', '',  ('gfbf', '2022b')),
```

Comment:
There is no need to specify the toolchain for SciPy-bundle, easybuild will resolve it alone